src/sort-filter-bar/sort-filter-bar.ts

```diff
@@ -959,7 +959,7 @@ export class SortFilterBar
         }
 
         #desktop-sort-selector li button.selected {
-          font-weight: bold;
+          text-shadow: rgb(0 0 0) 0.08rem 0 0;
```

Comment:
This _does_ address the text shifting problem, but I find the text shadows look noticeably different from the actual boldface font. I'm not sure inconsistent font appearances are a good tradeoff for fixing the slight shifts.

One alternative might be to place an always-bolded `visibility: hidden` element with the same content underneath each sort button, essentially reserving the needed space invisibly. Might take a bit of fiddling to get the positioning right, but that way we could keep the desired bold fonts. What do you think?

Reply:
Hi Laton!
Thanks for raising the question around font appearances. I applied another approach to keep using bold but handling the text shift problem is gone. Please go through the code changes and again do QA. Thanks!!